tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1131,3 +1131,9 @@ test_i2c_sts4x: sts4x@9d {
 	reg = <0x99>;
 	repeatability = <2>;
 };
```

Comment:
Looks like the reg property of the previous sensor binding is wrong, but obviously not your fault :)